packages/recommend/src/types/RecommendationsQuery.ts

```diff
@@ -39,3 +39,18 @@ export type RecommendationsQuery = {
    */
   readonly fallbackParameters?: RecommendSearchOptions;
 };
+
+/**
+ * Base type for models that don't require an `objectID`.
+ *
+ * Currently the only model that doesn't require an `objectID` is `recommended-for-you`.
+ */
+export type RecommendationsQueryWithoutObjectID = Omit<
+  RecommendationsQuery,
+  'objectID' | 'model'
+> & {
+  /**
+   * The name of the Recommendation model to use.
+   */
+  readonly model: 'recommended-for-you';
+};
```

Comment:
We don't seem to need this `RecommendationsQueryWithoutObjectID` type. In `RecommendedForYouQuery` we can also omit `objectID`, and use `RecommendedForYouQuery` in `getRecommendations()`?

Reply:
Good question, for `getRecommendedForYou` I also need to omit `model` from the parameters.

Reply:
Addressed feedback in last commit